open_tms_backend/open_tms_close_ticket/index.js

```diff
@@ -1,4 +1,43 @@
 import { shared } from '@appblocks/node-sdk'
+import { Ticket, TicketRevision, TicketActivity } from './models'
+
+const validateProcessPath = (currentStage, nextStage, assignee) => {
+  const processPath = {
```

Comment:
Here  we should get the process_path to evaluate if the next stage to which ticket is going is valid or not, from the database. Refer to prisma/schema.prisma to understand more about the models and relations.